index.html

```diff
@@ -0,0 +1,123 @@
+             </div>
+            </div>
+          </div>
+        </section>
+        <section class="main-bottom">
+            <h2 class="section-title">Main Program</h2>
+            <div class="main-bottom-in">
+                <div class="card">
+                    <div class="pro">
+                    <i class="fa-solid fa-address-card"></i>
+                    </div>
+                    <h2 class="title">Lectures</h2>
+                    <p>listen to the speakers from various countries talking about fitness</p>
+                 </div>
+                <div class="card">
+                    <div class="pro">
+                    <i class="fa-solid fa-phone-flip"></i>
+                    </div>
+                    <h2 class="title">caller</h2>
+                    <p>Meet your linda orls and talk to her on which topics you are interested to</p>
+                </div>
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-message"></i>
+                        </div>
+                    <h2 class="title">Forums</h2>
+                    <p>Share your thoughts and opinions with other fitness mentor. </p>
+                </div>
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-clock"></i>
+                        </div>
+                    <h2 class="title">Programs</h2>
+                    <p>After joining the program you have to dedicate 3+ hours of learning each day.</p>
+                </div>
+              
+                <div class="card">
+                    <div class="pro">
+                        <i class="fa-solid fa-people-group"></i>
+                        </div>
+                    <h2 class="title">Workshop</h2>
+                    <p>Try creating your own work using open license</p>
+                </div>
+            </div>
+
+            <a href="#" class="promore">SEE THE WHOLE PROGRAM</a>
+            <button type="button" class="probtn">Join Microverse</button>
+        </section>
+           <!-- features speakers -->
+        <div class="main-feature">
+           <h2 class="speakerhead section-title">Featured Speakers</h2>
+           <div class="grid">
+
+           </div>
+        </div>
+    </main>
+        <div class="partners">
+            <h2 class="partnerhead section-title">Partner</h2>
+            <div class="partner-logo">
+                <img src="asset/butter-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Conspicuous-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/fitbit-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/Muscle-Lab-800x600-1-removebg-preview.png" alt="">
+                <img src="asset/technogym-1-removebg-preview.png" alt="">
+            
+            </div>
+        </div>
+    
```

Comment:
- [ ] Background images are different and smaller. Kindly change it.
- [ ] All section has left and right space in the original. Kİndly fix it.

![image](https://user-images.githubusercontent.com/38635158/175398885-02613a5a-f4f4-48ce-ac95-2148cc463bbf.png)

___
ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/175398837-16a21ee0-a9bb-4014-a685-54f335895772.png)


### MOBİLE VERSİON
- [ ] It should have 2 speakers with more button.
![image](https://user-images.githubusercontent.com/38635158/175401095-8cb69d5a-5f59-4794-bbf6-a7b1b4a07b67.png)

ORIGINAL
![image](https://user-images.githubusercontent.com/38635158/175401266-c5b4c307-11ce-4307-bd1c-a59323dfbf68.png)

Reply:
@ahmetbozaci  this is optional and it is not required

Reply:
@cynthia-nwannah You are right.